lib/Runtime/Library/JavascriptError.cpp

```diff
@@ -371,7 +371,9 @@ namespace Js
 
         if (FACILITY_CONTROL == HRESULT_FACILITY(hr) || FACILITY_JSCRIPT == HRESULT_FACILITY(hr))
         {
+#ifndef __ANDROID__
```

Comment:
What is this change for? Does this function work xplat? It looks like its mostly getting windows specific errors so perhaps the entire if condition shouldn't disabled for Android, no?